modules/installation-azure-service-principal.adoc

```diff
@@ -2,6 +2,11 @@
 //
 // * installing/installing_azure/installing-azure-account.adoc
 // * installing/installing_azure/installing-azure-user-infra.adoc
+// * installing/installing_azure_stack_hub/installing-azure-stack-hub-user-infra.adoc
+
+ifeval::["{context}" == "installing-azure-stack-hub-user-infra"]
+:ash:
+endif::[]
 
 [id="installation-azure-service-principal_{context}"]
 = Creating a service principal
```

Comment:
Steps 6 on in this link are not very applicable for Azure Stack Hub

Reply:
I don't know the specifics of the rest of step 6, but the second and third bullet about mint and passthrough mode do not apply to ASH. 

Action item for me: that line about seeing the Cloud Credential Operator entry in the Red Hat Operators reference content is out of date.